client/src/Tools/_framework/ToolPanels/GeneralContentControls.tsx

```diff
@@ -397,6 +429,125 @@ export function GeneralContentControls({
         <input type="hidden" name="imagePath" value={imagePath ?? undefined} />
         <input type="hidden" name="_action" value="update general" />
         <input type="hidden" name="id" value={id} />
+
+        {!contentData.isFolder ? (
+          <FormControl>
+            <Flex flexDirection="column" width="100%" rowGap={6}>
+              <FormLabel mt="16px">Content Classifications</FormLabel>
+
+              <Accordion allowMultiple>
+                {contentData.classifications.map((classification, i) => (
+                  <AccordionItem key={`classification${i}`}>
+                    <HStack>
+                      <h2>
+                        <AccordionButton>
+                          <HStack flex="1" textAlign="left" direction={"row"}>
+                            ;<Text as="b">{classification.code}</Text>
+                            <Text fontSize={"small"} pt="2px">
+                              {classification.system.name}
+                            </Text>
+                          </HStack>
+                          <AccordionIcon marginLeft="7px" />
+                        </AccordionButton>
+                      </h2>
+                      <Spacer />
+                      <CloseButton
```

Comment:
Add an `aria_label` attribute and wrap with a `<Tooltip>` to give more information about what this close button is doing. Could be something like `Remove classification ${classification.code}`.